docs/helpers/theme-builder.md

```diff
@@ -1,17 +1,112 @@
 # Theme Builder
 
-:::warning
-**Note:** Work is in progress 🚧
-:::
+### Introduction
 
-We are currently working on this section to provide you with detailed and comprehensive information about how Acode plugins work. Please check back soon for updates!
+The `ThemeBuilder` class from the `acode` library provides a comprehensive solution for creating and customizing themes in your application. It offers granular control over various UI elements, colors, and styles.
```

Comment:
This paragraph can be improved, as this is a plugin api

Reply:
I just read it m yah it sucks let me rewrite it

Reply:
Check the update